### ISSUE TITLE
download address dead.

### ISSUE THREAD

**Issue by Zedzou** [AgileDrones/Blackbird-Dataset#28]
sorry, the download address seems to be dead, has it been updated.

**Comment by ianpegg-bc** [comment]
And looks like the rendering also relies on config files that are not included in the repo or docker image.

**Comment by varunmurali1** [comment]
Yes, we are aware that the download links are dead. They are moving from AWS to another solution and we will have new instructions for them asap.

**Comment by RBJin** [comment]
@varunmurali1 Do you have a planned date for updating the download links?

**Comment by zhouyizhuo** [comment]
> Yes, we are aware that the download links are dead. They are moving from AWS to another solution and we will have new instructions for them asap.

@varunmurali1 Sorry to interrupt.Do you have a planned date for updating the download links?🙂

**Comment by maulanaazhari** [comment]
Hi @varunmurali1, is there any update on the download links?

**Comment by RoboticsPolyu** [comment]
@varunmurali1 , Hi, is there any update on the download links?

**Comment by yufengsjtu** [comment]
Hi, has there been any update to the data download link?

**Comment by Ganathipan** [comment]
Hi @varunmurali1, is there any updates regarding the download link?
